api/src/test/java/com/epam/pipeline/manager/datastorage/providers/aws/s3/S3HelperTest.java

```diff
@@ -171,6 +182,32 @@ public void testRestoreFileShouldCopyFileAndThenDeleteTheOriginalFile() {
         verify(amazonS3).deleteObjects(argThat(hasPathsAndVersions(pathVersionMap)));
     }
 
+    @Test
+    public void testRestoreFolderWithExcludeListShouldNotRestoreExcludeFiles() {
+        helper.restoreFolder(BUCKET, OLD_PATH, presettingForRestoreFolderMethodTests(true,
+                null, Collections.singletonList("*.jpg")));
+        verify(amazonS3).deleteObjects(argThat(hasPathsAndVersions(
+                Collections.singletonMap(OLD_PATH + "/" + NEW_PATH + "/secondFile.png", VERSION))));
```

Comment:
Could you make these vars "/", "/secondFile.png", "*.jpg", "/firstFile.jpg" constants